extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/RuntimeBeanRepository.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.core;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.camel.spi.BeanRepository;
+
+public final class RuntimeBeanRepository implements BeanRepository {
```

Comment:
RuntimeBeanRepository being the only caller of BeanManagerHelper, is there a reason to keep them separate? I mean the invocations of BeanManagerHelper could be inlined here.

Reply:
I've inlined it but can we please be pragmatic an try to solve one problem at time ? 

BeanManagerHelper was there long before this PR and was used in very same way so to me it is unrelated to this PR.

Reply:
Yes, sorry, of course, that would be a valid reason to reject my proposal.